.github/workflows/example.yml

```diff
@@ -8,8 +8,28 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Checkout
-        uses: actions/checkout@v2
+      - uses: actions/checkout@v2
+
+      - name: Cache Qodana dependencies
+        uses: actions/cache@v2
+        with:
+          path: ~/work/_temp/_github_home/cache
```

Comment:
But below you are using `cache-dir: /github/home/qodana-cache`

Reply:
Fixed in the last commit